### PR TITLE
Client Encryption: Adds release version of Microsoft.Azure.Cosmos to Microsoft.Azure.Cosmos.Encryption.Custom

### PR DESCRIPTION
**Pull request by vipulvishal-ms**
## Description

Changing Microsoft.Azure.Cosmos package version from 3.26.0-preview to 3.32.3

## Type of change

- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

### CODE REVIEW

**Review by kirankumarkolli — DISMISSED** [review]

File: Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <AssemblyName>Microsoft.Azure.Cosmos.Encryption.Custom</AssemblyName>
     <RootNamespace>Microsoft.Azure.Cosmos.Encryption.Custom</RootNamespace>
     <LangVersion>$(LangVersion)</LangVersion>
-    <IsPreview>true</IsPreview>
+    <IsPreview>false</IsPreview>
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj:7]: Should this line be removed? Otherwise, when will the condition that uses this value as True be used?

Comment (vipulvishal-ms) [reply Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj:7]: we pass this value from command line and that value override this value

Comment (kr-santosh) [reply Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj:7]: This is not required if the intension was to make sure through direct proj reference with preview test (ENCRYPTIONPREVIEW) we are still compatible.

**Comment by ealsur** [comment]
@msftbot make sure to wait for @kr-santosh to approve

**Review by ealsur — APPROVED** [review]

File: Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj [hunk 2]
```diff
@@ -24,8 +22,12 @@
     <AdditionalFiles Include="..\..\Microsoft.Azure.Cosmos\src\stylecop.json" Link="stylecop.json" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(SdkProjectRef)' != 'True' ">
-    <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.26.0-preview" />
+  <ItemGroup Condition=" '$(SdkProjectRef)' != 'True' AND '$(IsPreview)' != 'True' ">
+      <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.32.3" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(SdkProjectRef)' != 'True' AND '$(IsPreview)' == 'True' ">
```
Comment (kr-santosh) [inline Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj:29]: not sure why is this required? Just remove the preview specific jobs.

Comment (vipulvishal-ms) [reply Microsoft.Azure.Cosmos.Encryption.Custom/src/Microsoft.Azure.Cosmos.Encryption.Custom.csproj:29]: one benefit of this way is if we want to again release any preview in future it will be just a config change.
